manifests/profile/master/postgres_access.pp

```diff
@@ -1,19 +1,19 @@
-# @summary Apply this class to a PE-managed postgres instance to allow access from telegraf
+# @summary Apply this class to a PE PostgreSQL node to allow access by Telegraf.
 #
 # @param telegraf_host
-#   The FQDN of the host where telegraf runs.  
-#   Defaults to an empty string.  You can explicitly set this parameter or the class attempts to lookup which host has the puppet_metrics_dashboard class applied in PuppetDB.  If the parameter is not set and the lookup does not return anything we issue a warning.
+#   The FQDN of the host running Telegraf. Defaults to an empty string.
+#   You can define this parameter, otherwise this class will query PuppetDB for a dashboard host.
 #
-# @example Allow access to PE-managed Postgres nodes
+# @example Apply this class to PE PostgreSQL nodes.
 #   class { 'puppet_metrics_dashboard::profile::master::postgres_access':
-#     telegraf_host => 'grafana-server.example.com',
+#     telegraf_host => 'dashboard.example.com',
 #   }
 #
 class puppet_metrics_dashboard::profile::master::postgres_access (
-  String $telegraf_host = ''
+  String $telegraf_host = "" # lint:ignore:double_quoted_strings
 ){
 
```

Comment:
The above prevents single quotes from being populated in the Console, which can lead to JSON errors.

Reply:
THe JSON errors if missed and using the console will bring PuppetDB (postgres) entirely down as the single quotes corrupt the pg_ident.conf file

Reply:
I think we are making this overly complicated... let's just change it to this: 

```puppet
Optional[String[1]] $telegraf_host = udef,
```

I have added a comment about how this would change the code that starts on line 18.

Reply:
OK